Sources/TABResourceLoader/Protocols/Resource types/NetworkPropertyListDecodableResourceType.swift

```diff
@@ -35,4 +35,11 @@ public extension NetworkPropertyListDecodableResourceType {
   var httpHeaderFields: [String: String]? {
     return ["Content-Type": "application/x-plist"]
   }
+
```

Comment:
i suggest adding `var jsonBody: Any? { get }` to the protocol definition. this demands that your conforming concrete type provides this property, and the property name is more descriptive of how it will be used.

Reply:
actually, this plist one is a bit of a mess. it's intended to parse a plist from the response. the type being sent in the request is JSON, as you've created below. but the Content-Type header field is "application/x-plist" - that's wrong. we should:
* either change the Content-Type to "application/json"
* change the transformation from `body` to `bodyData` to use PropertyListSerialization instead of JSONSerialization - although this is probably ludicrous
* change the entire protocol conformance pattern to distinguish between incoming and outgoing data formats (e.g. what if you want to send form data, but get a JSON response?)